src/desktop/src/ui/global/Migration.js

```diff
@@ -67,20 +88,51 @@ class Migration extends React.Component {
         ];
     }
 
+    changeNode = () => {
```

Comment:
We could just do: `this.props.setFullNode(...sample(this.props.nodes));` with `lodash/sample` here

Reply:
Desktop does not have `lodash` as dependency and additionally this method checks to not set the node that is already set.